system_setup/client/controllers/summary.py

```diff
@@ -87,10 +89,21 @@ async def _wait_status(self, context):
 
     async def make_ui(self):
         real_name = ""
+
+        tmp_variant = "wsl_setup"
+        onsite_variant = getattr(self.app, "variant")
+        if onsite_variant is not None:
+            tmp_variant = onsite_variant
+
         identity = getattr(self.app.client, "identity")
         if identity is not None:
             data = await identity.GET()
             real_name = data.realname
+        if tmp_variant == "wsl_configuration":
+            tmp_uid = os.getuid()
+            if tmp_uid >= 1000 and tmp_uid < 65534:
+                real_name = pwd.getpwuid(tmp_uid)[0]
```

Comment:
I think we shuld rather change the summary message without using the user name here. Then the whole variant os.getuid is not needed here as we don’t need real_name.

Reply:
The logic is changed for this, needs check whether should be improved.

Reply:
I don’t see any changes there, please have a look before I make a second review pass.

Reply:
Sorry for that, fixed now